stan/math/prim/mat/fun/cov_exp_quad.hpp

```diff
@@ -84,8 +86,8 @@ namespace stan {
                   Eigen::Dynamic, Eigen::Dynamic>
     cov_exp_quad(const std::vector<T_x1>& x1,
                  const std::vector<T_x2>& x2,
-                 T_sigma& sigma,
-                 T_l& l) {
+                 const T_sigma& sigma,
+                 const T_l& l) {
       using std::exp;
```

Comment:
I guess someone put that there (maybe me?) to bring
it into the namespace.  Looks like we have our own
log(), too.  Who knew?  But yes, you won't need 
the using statement for that if it's included as it's
in the same namespace.